lib/dataSources.js

```diff
@@ -50,9 +51,12 @@ const update = (conn, name, options) => {
   }).then(httpBody);
 };
 
-const remove = (conn, name) => {
+const remove = (conn, name, params = {}) => {
   const headers = conn.headers();
-  return fetch(conn.request('admin', 'data_sources', name), {
+  const queryParams = Object.keys(params).length
+    ? `?${qs.stringify(params)}`
+    : '';
+  return fetch(conn.request('admin', 'data_sources', name, queryParams), {
```

Comment:
I'm not sure about this since it's adding an extra slash between the name and query params: 
`http://localhost:5820/admin/data_sources/MyDataSource/?force=true`

Reply:
yes, it keeps the ending slash but does work as expected

Reply:
we can append to the data source name to make it look better

Reply:
i think that would be preferable (and seems to be consistent with the rest of the uris we generate)

Reply:
It looks like we are doing it both ways. There is a `suffix` as the last argument and some in places appending.

Reply:
actually it looks like those `suffix`'s have names attached.. just kidding